service/src/main/okapi/tenant/sample_data/_data.groovy

```diff
@@ -102,13 +102,13 @@ try {
                               key: 'wms_connector_password'
                               ).save(flush:true, failOnError: true);
 
-  AppSetting wms_connector_password = AppSetting.findByKey('system_base_url') ?: new AppSetting( 
+  AppSetting system_base_url = AppSetting.findByKey('system_base_url') ?: new AppSetting( 
```

Comment:
Just checking you meant to _replace_ these old settings, rather than adding the new ones.

Reply:
The title was incorrectly the same as another one, meaning they all failed to build. There shouldn't even be an "old setting" in any running instances, I think.

Reply:
Cool, thanks!